### DISCUSSION TITLE
Where to specify url external CardDAV server ?

### DISCUSSION THREAD

**Discussion by user9931** [the-djmaze/snappymail#1856] [Q&A]
On the "contacts" tab in admin panel there's a button "Allow contacts sync (with external CardDAV server)". But where to specify url to this "external carddav server"? sorry if its obviouse, cant find it in docs

**Comment by cm-schl** [comment]
If you enable this option in the admin panel the user himself can specify the url in his personal settings.
See: https://github.com/the-djmaze/snappymail/wiki/Admin-Manual#menu-contacts (😉).

**Comment by user9931** [comment]
Thank you, I missed this manual somehow :) but it's really not I'm looking for.  So this option just allow user to specify carddav server by themselves, and  dont  allow admin to specify this server for all users in admin ui?

**Comment by cm-schl** [comment]
As far as I know no - but if you're using Nextcloud there is a PR waiting for approval that probably is solving your problem: https://github.com/the-djmaze/snappymail/pull/1227

**Comment by user9931** [comment]
I'm not using nextcloud , but this indeed can help, at least as starting point, thanks for pointing out
